rfcs/20220201-quantization-scaling/README.md

```diff
@@ -0,0 +1,305 @@
+Proposal for updating quantization scale functionality in oneDNN
+================================================================
+
+## 0. Motivation
+
+There are multiple requests for extending scaling support in oneDNN primitives:
+- run-time scaling in Eltwise post-op;
+- scaling for src1 in Binary post-op;
```

Comment:
Depthwise post-op is not covered but it supports only statically provided scales (no runtime support) and it takes it from API. Guess, we need some text and plan for it as well. Thank you.

Reply:
Added it into a table of currently supported scale-mechanisms. It is similar to other post-ops with a built-in scale parameter, which are covered in the RFC already.

Reply:
So how could we use Matmul + binary_add full pipeline int8? I think we need both src0 scale and src1 scale in this case. The equation is as follows, we assume src0=i_s8*w_s8, post_s8 is the output we aim to get: 
src0_s8 = src0*src0_scale,  
src1_s8 = src1*src1_scale
post_s8 = binary_add(src0_s8, src1_s8) * post_op_scale